setup.sh

```diff
@@ -75,6 +75,8 @@ kubectl --namespace crossplane-system \
     --from-literal accessKeyID=$AWS_ACCESS_KEY_ID \
     --from-literal secretAccessKey=$AWS_SECRET_ACCESS_KEY
 
+kubectl apply --filename crossplane-config/deployment-runtime-config.yaml
```

Comment:
I noticed that we actually never applied the runtime config in the setup script (seems like I forgot about it when adding the file)